cumulus/polkadot-parachain/Cargo.toml

```diff
@@ -15,19 +14,14 @@ name = "polkadot-parachain"
 path = "src/main.rs"
 
 [dependencies]
-async-trait = { workspace = true }
-clap = { features = ["derive"], workspace = true }
-codec = { workspace = true, default-features = true }
 color-eyre = { workspace = true }
-color-print = { workspace = true }
-futures = { workspace = true }
 hex-literal = { workspace = true, default-features = true }
 log = { workspace = true, default-features = true }
 serde = { features = ["derive"], workspace = true, default-features = true }
 serde_json = { workspace = true, default-features = true }
-docify = { workspace = true }
 
 # Local
+polkadot-parachain-lib = { workspace = true }
```

Comment:
ideally, this file should not have any deps to polkadot-sdk crates other than `polkadot-parachain-lib`. deps like sp-runtime etc should be reexported via polkadot-parachain-lib

the goal is to minimize API surface area to use polkadot-parachain-lib to improve DX and chance of breaking changes

Reply:
ok most of the deps are for dev chain spec generation, in that case are fine. the only type that I find needed reexport is `sc-chain-spec::ChainSpec`

Reply:
Makes sense. I re-exported `sc-chain-spec::ChainSpec`